### PR TITLE
changes to label group component design guidelines

### PR DESCRIPTION
**Pull request by pankajshivpuje**
Link to the updated Label group [design doc]

Here are the updated images:

(https://docs.google.com/document/d/1mSQlcssmhwOuiByR3Hx_5zfbZcTCzuH9DBs2oKqpdcg/edit#)
editable-label-group-in-a-table_1
editable-label-group-in-a-table_2_1
editable-label-group-in-a-table_2
editable-label-group-in-a-table_3_1
editable-label-group-in-a-table_3_2
editable-label-group-in-a-table_3
editable-label-group-in-a-table_4_1_add-label-model
editable-label-group-in-a-table_4_2
editable-label-group-in-a-table_4

### CODE REVIEW

File: packages/v4/patternfly-docs/content/design-guidelines/components/label-group/label-group.md [hunk 1]
```diff
@@ -26,3 +26,40 @@ Label groups should be used when there are multiple labels assigned to a compone
 
 ### Label groups in a description list
 <img src="./img/description-list.png" alt="Description list with labels" width="520"/>
+
+## Variations
+
+### Editable label group
+
+#### Adding a new label to a group, without a group label
```
Comment (mcarrano) [inline packages/v4/patternfly-docs/content/design-guidelines/components/label-group/label-group.md:34]: I'm a bit confused about this variation. The title says "adding a new label," but the example seems to be about editing a label. I'd also spell out the word "one" here, as in "...there is a need to edit one or more labels."

**Review by mcarrano — CHANGES_REQUESTED** [review]
@pankajshivpuje this is looking good except for a couple of minor things. See my comments. I also think that it would help to use a callout to point to the action in some of the examples since it was hard to tell at a glance where one would click.

I know you will be out after this week, but if you make these changes today I will merge this as @mmenestr ison PTO until Jan 9.

**Comment by pankajshivpuje** [comment]
@mcarrano I have updated the design guidelines doc, added an image and made textual changes to address your questions. I have also pushed the updated in the PR. Let me know if any additional changes are needed.

File: packages/v4/patternfly-docs/content/design-guidelines/components/label-group/label-group.md (hunk 1 above)
Comment (pankajshivpuje) [reply packages/v4/patternfly-docs/content/design-guidelines/components/label-group/label-group.md:34]: @mcarrano I have updated the design guidelines doc, added an image and made textual changes to address your questions. I have also pushed the updated in the PR. Let me know if any additional changes are needed.

**Review by mcarrano — APPROVED** [review]
Looks great. Thanks for making those changes @pankajshivpuje . I will merge.

**Pull request merged**